services/minespace-web/src/components/dashboard/mine/variances/VarianceDetails.js

```diff
@@ -22,57 +22,42 @@ const defaultProps = {
 };
 
 export const VarianceDetails = (props) => {
-  const isApproved =
-    props.variance.variance_application_status_code === Strings.VARIANCE_APPROVED_CODE;
+  // const isApproved =
```

Comment:
Is this still needed?

Reply:
I'm thinking that it might be, come this Sprint. I had to comment it out to get the linter to not complain about an unused variable at the time.